include/h2o.h

```diff
@@ -212,13 +212,13 @@ struct st_h2o_pathconf_t {
      */
     H2O_VECTOR(h2o_handler_t *) handlers;
     /**
-     * list of filters
+     * list of filters that will be copied into req->filters
```

Comment:
Comment like "list of filters to apply (unless when processing a subrequest)" might better clarify the role (than what is being done).